engines/agi/opcodes.cpp

```diff
@@ -462,6 +462,11 @@ void AgiEngine::setupOpCodes(uint16 version) {
 		}
 	}
 
+	// AGIMOUSE games use an modified push.script that updates mouse state
+	if ((getFeatures() & GF_AGIMOUSE) && version >= 0x2000) {
```

Comment:
Why do we need a version check here? Shouldn't the AGIMOUSE flag be sufficient?

Reply:
I thought it would clarify but clearly it didn't! =)

Also now I see that even in the event of an incorrect detection entry, this would get handled by the subsequent code, so I'll remove it.